topics/statistics/tutorials/fruit_360/tutorial.md

```diff
@@ -0,0 +1,419 @@
+3 pooling layers are MaxPool layers with pool size of 2 x 2. Afterwards, we flatten the previous layer's 
+output (every row/colum/channel would be an individual node), then add a fully connected layer with 256 
+nodes and relu activation function. Finally, we add a fully connected layers with 10 nodes, and use 
+softmax activation function to get the probability of each fruit/vegetable. Fruit/vegetable with the 
+highest probability is predicted by CNN. The model config can be downloaded as a JSON file.
+
+### **Create a deep learning model**
+
+> ### {% icon hands_on %} Hands-on: Model builder (Optimizer, loss function, and fit parameters)
+>
+> - {% tool [Create deep learning model](toolshed.g2.bx.psu.edu/repos/bgruening/keras_model_builder/keras_model_builder/0.5.0) %}
+>    - *"Choose a building mode"*: `Build a training model`
+>    - *"Select the dataset containing model configuration"*: Select the *Keras Model Config* from the previous step.
+>    - *"Do classification or regression?"*: `KerasGClassifier`
+>    - In *"Compile Parameters"*:
+>        - *"Select a loss function"*: `categorical_crossentropy`
+>        - *"Select an optimizer"*: `Adam - Adam optimizer `
+>        - *"Select metrics"*: `acc/accuracy`
+>    - In *"Fit Parameters"*:
+>        - *"epochs"*: `40`
+>        - *"batch_size"*: `50`
+>    - Click *"Execute"*
+{: .hands_on}
+
+A loss function measures how different the predicted output is from the expected output. For multi-class classification problems, 
+we use *categorical cross entropy* as loss function. Epochs is the number of times the whole training data is used to train the 
+model. Setting *epochs* to 40 means each training example in our dataset is used 40 times to train our model. If we update network 
+weights/biases after all the training data is feed to the network, the training will be very slow (as we have 5014 training examples 
+in our dataset). To speed up the training, we present only a subset of the training examples to the network, after which we update 
+the weights/biases. *batch_size* decides the size of this subset. The model builder can be downloaded as a zip file.
+
+### **Deep learning training and evaluation**
+
+> ### {% icon hands_on %} Hands-on: Training the model
+>
+> - {% tool [Deep learning training and evaluation](toolshed.g2.bx.psu.edu/repos/bgruening/keras_train_and_eval/keras_train_and_eval/1.0.8.2) %}
+>    - *"Select a scheme"*: `Train and Validate`
+>    - *"Choose the dataset containing pipeline/estimator object"*: Select the *Keras Model Builder* from the previous step.
+>    - *"Select input type:"*: `tabular data`
+>        - *"Training samples dataset"*: Select `train_X_10` dataset
+>        - *"Choose how to select data by column:"*: `All columns`
+>        - *"Dataset containing class labels or target values"*: Select `train_y_10` dataset
+>        - *"Choose how to select data by column:"*: `All columns`
```

Comment:
Should we select all columns from the output `train_y_10` dataset? Or should we only take the `Label` column (3rd column) for training? Similar comment would be for `Model prediction` step. Also, we should also add information about `Does the dataset contain header:` option from the tool. Should we set it as true because datasets seem to contain headers. Thanks!

Reply:
Actually, I have to add new steps here to run One Hot Encoding on the label in train_y_10. Will revise and push the change shortly. Thanks @anuprulez!

Reply:
@anuprulez I just pushed the updated tutorial. It has a section on how to calculate the OHE representation of training labels. Thanks!

Reply:
Ok, will have a look :)